types/mongodb/index.d.ts

```diff
@@ -1294,6 +1294,7 @@ export interface CollectionAggregationOptions {
      * Allow driver to bypass schema validation in MongoDB 3.2 or higher.
      */
     bypassDocumentValidation?: boolean;
+    hint?: string | Object;   
```

Comment:
Did you mean `object`?

Reply:
It seems to be a convention in this file to use `Object` for an object of unknown structure (but not  `number`, `string`, `undefined`,etc..)

Reply:
We generally recommend against `Object`: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

If you're going to stick with `Object`, I think `string` is redundant since `Object` doesn't exclude primitives.

Reply:
Yeah, it's not something I normally use, but it's used everywhere else in the file so I went with convention.  Maybe I should just set it to `any` ?

Reply:
Well, I changed it to `string | object`.  Maybe it'll start a trend.